renderer/components/FormattedMarkdownMessage.js

```diff
@@ -35,23 +35,16 @@ const remarkReactComponents = {
   a: OpenLinkInBrowser
 }
 
-class FormattedMarkdownMessage extends React.Component {
-  render() {
-    const { formatMessage } = this.context.intl
-    const { id, defaultMessage, values, description } = this.props
-    const messageDescriptor = { id, defaultMessage, description }
-    const message = formatMessage(messageDescriptor, values)
-    // When the message is the ID means it's not defined and we just emit not
-    // element
-    if (message === id) {
-      return null
-    }
-    return remark().use(reactRenderer, {remarkReactComponents}).processSync(message).contents
+const FormattedMarkdownMessage = ({ id, defaultMessage, values, description }) => {
+  const { formatMessage } = useIntl()
+  const messageDescriptor = { id, defaultMessage, description }
+  const message = formatMessage(messageDescriptor, values)
+  // When the message is the ID means it's not defined and we just emit not
+  // element
```

Comment:
what does "not element" mean?

Reply:
I suspect this is a typo and can be rephrased as:

> When the message is the ID means it's not defined, so we don't emit an element

I do wonder, though if this is desired behaviour. It may be useful in these cases to emit a special element which puts a placeholder in the DOM so that it's possible to see what are the strings that still need to be populated with items.

@sarathms what do you think?

Reply:
Good catch. I overlooked it. I don't think it is a good idea to return `null`. Best would be to just render the undefined key as-is which then aligns with the default behavior of `<FormattedMessage>`. Typically, our keys are pretty long and will most likely stick out enough to be noticeable.

Moreover, we can adapt the script we wrote for explorer which detects keys defined but not translated and stale keys in the csv.